addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting4Message.java

```diff
@@ -74,43 +80,56 @@ public static SubType fromByte(int input) throws RFXComUnsupportedValueException
     }
 
     public enum Commands {
-        UNDEFINED_0(0),
-        ON(1),
-        UNDEFINED_2(2),
-        UNDEFINED_3(3),
-        OFF(4);
+        OFF_0(0, false),
+        ON_1(1, true),
+        OFF_2(2, false),
+        ON_3(3, true),
+        OFF_4(4, false),
+        ON_5(5, true),
+        ON_7(7, true),
+        ON_9(9, true),
+        ON_12(12, true),
+        UNKNOWN(-1, false);
```

Comment:
Ah right, so you're saying that they are likely to exist, but because we don't have a spec, we can't say one way or the other.

If it gets set to unknown, can it still be used? If it can't, then I think we should be throwing an error. You know I don't like undefined behaviour, and if it throws an error and gets reported, it can be added to the code properly.

Is this one of the values that gets discovered? If we did add the other 50, could the user then decide whether it was ON or OFF in the Paper UI somehow?

Either way, we could put out a warning message in the log telling them to do something, like "report to github" or "go to the paper UI and change X".